src/main/java/org/terasology/ligthandshadow/logic/CardSystem.java

```diff
@@ -136,26 +137,11 @@ public void placeCard(ActivateEvent event, EntityRef entity) {
 
     @ReceiveEvent(components = {CardComponent.class, LocationComponent.class})
     public void onDamaged(OnDamagedEvent event, EntityRef entity) {
-        LocationComponent location = entity.getComponent(LocationComponent.class);
-        CardComponent cardComponent = entity.getComponent(CardComponent.class);
-        Vector3f center = location.getWorldPosition();
-        EntityRef particlesEntity = entityManager.create();
-        particlesEntity.addComponent(new LocationComponent(center));
-
-        BlockParticleEffectComponent particleEffect = new BlockParticleEffectComponent();
-        particleEffect.spawnCount = 64;
-        particleEffect.blockType = cardComponent.bottomBlockFamily.getArchetypeBlock().getURI().toString();
-        particleEffect.initialVelocityRange.set(4, 4, 4);
-        particleEffect.spawnRange.set(0.3f, 0.3f, 0.3f);
-        particleEffect.destroyEntityOnCompletion = true;
-        particleEffect.minSize = 0.05f;
-        particleEffect.maxSize = 0.1f;
-        particleEffect.minLifespan = 1f;
-        particleEffect.maxLifespan = 1.5f;
-        particleEffect.targetVelocity.set(0, -5, 0);
-        particleEffect.acceleration.set(2f, 2f, 2f);
-        particleEffect.collideWithBlocks = true;
-        particlesEntity.addComponent(particleEffect);
+        //TODO change particle texture to match block texture
+        Prefab particleEffectPrefab = prefabManager.getPrefab("LightAndShadowResources:cardParticleEffect");
+        for (Component c : particleEffectPrefab.iterateComponents()) {
+            entity.addComponent(c);
```

Comment:
Right, most component should only be added once, despite the `ParticleEmitterComponent` itself. AFAIK it will just emit a fixed number of particles and then it will be destroyed after "dying". I'll look into this later.